types/instance.d.ts

```diff
@@ -82,6 +82,7 @@ export interface FastifyInstance<
 
   register: FastifyRegister<FastifyInstance<RawServer, RawRequest, RawReply, Logger> & PromiseLike<undefined>>;
 
+  routing(req: RawRequest, res: RawReply): void;
```

Comment:
There is an optional third parameter

https://github.com/delvedor/find-my-way//#lookuprequest-response-context

Reply:
It do not have third param because we do not pass it.
`httpHandler` is `findMyWay.lookup`.
https://github.com/fastify/fastify/blob/13f2c36c79877428dab43581f7b8574a9b180b47/fastify.js#L663-L680